### DISCUSSION TITLE
How to visualize capillary pressure in Paraview

### DISCUSSION THREAD

**Discussion by javierpagalo** [PMEAL/porespy#598] [Q&A]
Hello
I'm undergraduate student and I use paraview for visualization of porous media.
I would like to know if there is a way to see the capillary pressure of a media like this in paraview. 
![48413f5a-d006-474a-bd16-d93665956c03](https://user-images.githubusercontent.com/46061502/141375521-4badfb3a-f219-4029-ba48-a210fd1babd8.png)

**Comment by jgostick** [comment]
I think I'll need a little more info.  Here is my understanding so far. You have a voxel image and you wish to visualize the capillary pressure at which each voxel will be invaded at. If so, then you'll want to use the porespy.filters.porosimetry function.  The result is that each voxel is filled with the capillary radius at which it was invaded.  You can convert to this pressure by applying the Young-Laplace equation to each pixel with your specific fluid properties.  This can then be opened in paraview and colored according the voxel values.  The result only looks good if you threshold the voxels to only show which voxels were invaded below a certain value of Pc.  Then you can see the fluid configurations.  If this is off base, please provide additional info and I'll try again.  Porosimetry is a special case of the more general local thickness filter discussed here: https://porespy.org/examples/filters/tutorials/local_thickness.html.
